web/app/scripts/src/libraries.es.js

```diff
@@ -0,0 +1 @@
+export const angular = window.angular;
```

Comment:
It actually does in the `npm` package. I thought of using it this way, however there are some compatibility issues:
* If you require a module it will be browserified, probably there is a way to tell browserify to ignore some files, but then in a package that will use this library es6 will be required to browserify it too. Also other packages might need updating to work in the same way. It would be probably quite painful to get it working.